tests/models/test_tpu.py

```diff
@@ -289,33 +279,6 @@ def test_broadcast(rank):
     xmp.spawn(test_broadcast, nprocs=8, start_method="fork")
 
 
-@pytest.mark.parametrize(
```

Comment:
Tests verifies `tpu_id` is removed, since this internal property is removed in this PR. 

The tpu_cores mis-configuration tests are merged with `test_tpu_misconfiguration()`

Reply:
what about the cases where error is not expected? are we testing them somewhere?

Reply:
then it's good to go 🚀